### ISSUE TITLE
Always Authorizing

### ISSUE THREAD

**Issue by dleszcynski** [VitaliyRodnenko/geeknote#378]
I have authorized Geeknote, set it up correctly without error, set up Two Factor Authentication, logged in correctly using 'geeknote login' and still whenever I write 'geeknote notebook-list' it just takes me back through the login procedure. Same thing with 'geeknote create --title Test'. This behavior happens in both Terminal.app and in iTerm2.app. I'm using zsh. I'm running macOS Siera 10.12.2 on a 15-inch MacBook Pro Retina Mid 2014.

**Comment by wangsen992** [comment]
Yes I have the same issue here, using a mac air 2013 I think. Same problem. Wrong login input will cause warning, but correct one shows nothing, and not logged in.

**Comment by dustinkeib** [comment]
I also had this issue.  PR374 https://github.com/VitaliyRodnenko/geeknote/pull/374 appears to fix this for me.

**Comment by quydo** [comment]
The same for me, I use 2step authentication :D

**Comment by quydo** [comment]
this issue was fixed this problem

**Comment by danmichaelo** [comment]
Or use this fork: https://github.com/jeffkowalski/geeknote , which is ~500 commits ahead of this and includes the fix by pipakin

**Comment by dustinkeib** [comment]
Thank you Dan, great point!
